config/geantSim.py

```diff
@@ -68,10 +69,12 @@
 
 
 from Gaudi.Configuration import *
+from GaudiKernel import SystemOfUnits as units
 ##############################################################################################################
 #######                                         GEOMETRY                                         #############
 ##############################################################################################################
-path_to_detector = '/afs/cern.ch/work/h/helsens/public/FCCsoft/FCCSW-0.8.3/'
+#path_to_detector = '/afs/cern.ch/work/h/helsens/public/FCCsoft/FCCSW-0.8.3/'
+path_to_detector = "/afs/cern.ch/work/v/vavolkl/public/fcc.cern.ch/sw/pre0.9.1/FCCSW/InstallArea/"
```

Comment:
can you change that to the official installed released? It pointed to my local due to the non installation of the tail catcher geo